src/Post.php

```diff
@@ -63,20 +63,27 @@ public function __construct($pid = null)
      */
     public function set_data_layer(): void
     {
+        $is_new_ia = !empty(planet4_get_option('new_ia'));
         if (is_front_page()) {
             $this->data_layer['page_category'] = 'Homepage';
-        } elseif ($this->is_act_page()) {
+        } elseif (!$is_new_ia && $this->is_act_page()) {
             $this->data_layer['page_category'] = 'Act';
-        } elseif ($this->is_explore_page()) {
+        } elseif (!$is_new_ia && $this->is_explore_page()) {
             $this->data_layer['page_category'] = 'Explore';
-        } elseif ($this->is_take_action_page()) {
+        } elseif (!$is_new_ia && $this->is_act_page_child()) {
             $this->data_layer['page_category'] = 'Take Action';
-        } elseif ($this->is_issue_page()) {
+        } elseif (!$is_new_ia && $this->is_issue_page()) {
             $this->data_layer['page_category'] = 'Issue Page';
         } elseif ($this->is_campaign_page()) {
             $this->data_layer['page_category'] = 'Campaign Page';
         } elseif (is_tag()) {
             $this->data_layer['page_category'] = 'Tag Page';
+        } elseif ($is_new_ia && $this->is_get_informed_page()) {
+            $this->data_layer['page_category'] = 'Get Informed';
+        } elseif ($is_new_ia && $this->is_take_action_page()) {
+            $this->data_layer['page_category'] = 'Take Action';
```

Comment:
I'm waiting for Molly to confirm this value, since it's also the one we use without the new IA for the Act page children